.github/workflows/playwright.yml

```diff
@@ -0,0 +1,50 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example: ["pets"]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "pnpm"
+
+      - name: Pnpm Build
+        run: |
+          pnpm install
+          pnpm turbo build;
+        working-directory: ./examples/${{ matrix.example }}
+
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
+        working-directory: ./examples/${{ matrix.example }}
+
+      - name: Run Playwright tests
+        run: pnpm exec playwright test
+        working-directory: ./examples/${{ matrix.example }}
+
+      - uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: ${{ matrix.example }}-playwright-report
+          path: ./examples/${{ matrix.example }}/playwright-report/
+          retention-days: 30
```

Comment:
Added artifact upload on failure, to be able to check the traces and understand what's wrong.

Example of report: https://github.com/gardencmp/jazz/actions/runs/10646563739/artifacts/1877985414